java/src/main/native/OrtJniUtil.c

```diff
@@ -1024,6 +1029,9 @@ void checkOrtStatus(JNIEnv *jniEnv, const OrtApi * api, OrtStatus * status) {
         const char* message = api->GetErrorMessage(status);
         size_t len = strlen(message)+1;
         char* copy = malloc(sizeof(char)*len);
+        if (copy == NULL) {
+          throwOrtException(jniEnv, 1, "Not enough memory");
```

Comment:
does code still execute after throwOrtException()?
if so, then the memcpy that uses a NULL `copy` is probably wrong. if not, then api->ReleaseStatus() won't get called.

Reply:
No. I think it won't.

Reply:
but C doesn't have exceptions, so I'm curious about what's happening here...

Reply:
You're right. I'll add "return" there.

Reply:
ok, I think adding return should work - looks like the status gets released now too so that's good.
will probably need to check other places where throwOrtException was added to verify we don't use the NULL result and that we clean things up.

Reply:
This just informs JVM that there was an exception. The function returns.